preprocess/coreference_resolution.py

```diff
@@ -15,22 +15,28 @@ def no_preprocess(news_vendor: str, object_name: str, input_csv_name: str,
 
 def coref_preprocess(news_vendor: str, object_name: str, input_csv_name: str,
                      output_dir: str = "data/preprocessed_data"):
-    nlp = spacy.load("en_coreference_web_trf")
+
     df = pd.read_csv(f'{DATA_DIR}/{input_csv_name}')
     output_filename = f'{news_vendor}_{object_name}_{PreprocessNames.COREF}.csv'
     full_output_path = f'{output_dir}/{output_filename}'
-    if not os.path.isfile(full_output_path):
+
+    if os.path.isfile(full_output_path):
+        df = pd.read_csv(full_output_path)
```

Comment:
@idobiran11 
you see what i did here? it's ok right?

Reply:
ya good